infra/blueprint-test/pkg/tft/terraform.go

```diff
@@ -205,6 +210,19 @@ func (b *TFBlueprintTest) getTFOutputsAsInputs(o map[string]interface{}) map[str
 	return n
 }
 
+// loadSetupSAKey activate a gcp encoded sa key
+func (b *TFBlueprintTest) loadSetupSAKey(saKey string, isEncoded bool) {
```

Comment:
Instead of having the isEncoded logic here, I'd just keep it on line 164.

Reply:
I thought it would be cleaner in the _func loadSetupSAKey_. 

Well, I'm going to delete the func because it will lose the sense of existing just to call gcloud cmd.